tasks/fr/insee.py

```diff
@@ -277,13 +277,28 @@ def columns(self):
             type='Text',
             name='DCOMIRIS',
             description='Full Code IRIS. Result of the concatenation of DEPCOM and IRIS attributes. ',
-            weight=0,
+            weight=1,
```

Comment:
Sounds like an old one leaked in, yup let's take a look at the diff once the French work is merged!